modules/openapi-generator/src/main/resources/JavaSpring/beanValidationCore.mustache

```diff
@@ -11,8 +11,10 @@ minLength not set, maxLength set
 }}{{#minItems}}{{^maxItems}}@Size(min = {{minItems}}) {{/maxItems}}{{/minItems}}{{!
 @Size: minItems not set && maxItems set
 }}{{^minItems}}{{#maxItems}}@Size(max = {{.}}) {{/maxItems}}{{/minItems}}{{!
-@Email: useBeanValidation set && isEmail set
-}}{{#useBeanValidation}}{{#isEmail}}@Email{{/isEmail}}{{/useBeanValidation}}{{!
+@Email: useBeanValidation
+}}{{#isEmail}}{{#useBeanValidation}}@javax.validation.constraints.Email{{/useBeanValidation}}{{!
```

Comment:
what about jakarta? assuming current imports in model.mustache I assume that you can use only `@Email`(without package name) when we use javax/jakarta annotation, while for Hibernate you should use package + annotation

Reply:
gotcha! gonna fix this, yes